source/content/guides/decoupled/wp-nextjs-frontend-starters/03-caching.md

```diff
@@ -0,0 +1,178 @@
+anchorid: caching
+contenttype: [guide]
+innav: [true]
+categories: [create]
+cms: [decoupled]
+audience: [development]
+product: [decoupled]
+integration: [--]
+---
+
+Cache settings are configured out-of-the-box for you, but there are circumstances in which you might need to make your own configurations, for example, your CMS is not on Pantheon but your frontend is on Pantheon.
+
+## How it Works
+
+Edge caching can improve performance for Front-End Sites that rely on API endpoints, and also reduce the load on your CMS in cases when a large number of API requests are made in a short period of time, such as during a full site build process.
+
+<Partial file="decoupled-caching.md" />
+
+## Using the Starter Kit
+
+The information in this section uses WordPress with the plugins below installed:
+
+- [WPGraphQL plugin](https://wordpress.org/plugins/wp-graphql/),
+- [WPGraphQL Smart Cache plugin](https://github.com/wp-graphql/wp-graphql-smart-cache)
+- [Pantheon Advanced Page Cache plugin](https://wordpress.org/plugins/pantheon-advanced-page-cache/)
+
+
+The `GraphqlClientFactory` class from our `@pantheon-systems/wordpress-kit` npm package adds the `Fastly-Debug` header to each request and uses the GET method by default to take advantage of WPGraphQL Smart Cache network caching. Responses from WordPress will contain the `Surrogate-Key` header. You can use these keys to instruct your frontend to purge content from a cache when the content in WordPress changes.
+
+## Cache-Control Headers
+
+This section explains how to set Cache-Control headers using
+`@pantheon-systems/wordpress-kit` in the `next-wordpress-starter`, or any
+Next.js application using the `wordpress-kit`.
+
+The `@pantheon-systems/wordpress-kit` npm package exports a function,
+`setEdgeHeaders`, which takes in a response object and a cache-control header
+value. The value is then set to the response object's headers so that when the
+request is sent along it will be cached at the edge.
+
+The default cache-control header is the following:
+
+```http
+Cache-Control: public, s-maxage=600
+```
+
+### Override WordPress Kit's Default Cache-Control Headers
+
+You must pass in your own cache-control header to override the default. For example:
+
+```jsx
+import { setEdgeHeaders } from '@pantheon-systems/wordpress-kit';
+
+export default function MyPage(props) {
+	// Page component here...
+}
+
+export async function getServerSideProps(context) {
+	// the response object from the server context
+	const { res } = context;
+
+	// setEdgeHeaders accepts an optional string which is a cache-control header
+	const myCacheControlHeader = 'public, max-age=604800, must-revalidate';
+
+	// Call setEdgeHeaders with the res object and your desired cache-control header
+	setEdgeHeaders({ res, cacheControl: myCacheControlHeader });
+
+	// Fetch data and return props...
+}
+```
+
+You can take advantage of this feature without using the starter kit.
+
+## Purge Edge Caching
```

Comment:
I'd expect the WP and Drupal versions of this entry to use the same heading. The Drupal one is 'Purge Caching'. This one reads better. Or alternatively something like 'Cache Purging'

Reply:
@backlineint Thanks!! I agree it reads better. Just pushed the update.